aws/table_aws_securitylake_data_lake.go

```diff
@@ -0,0 +1,93 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/securitylake"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsSecurityLakeDataLake(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_securitylake_data_lake",
+		Description: "AWS Security Lake Data Lake",
+		List: &plugin.ListConfig{
+			Hydrate: getSecurityLakeDataLake,
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "encryption_key",
+				Description: "The type of encryption key used by Security Lake to encrypt the lake configuration.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "replication_role_arn",
+				Description: "Replication settings for the Amazon S3 buckets. This parameter uses the IAM role created by you that is managed by Security Lake, to ensure the replication setting is correct.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "s3_bucket_arn",
+				Description: "Amazon Resource Names (ARNs) uniquely identify Amazon Web Services resources.",
```

Comment:
Is the description correct?

Reply:
The same description is mentioned in doc.
https://docs.aws.amazon.com/security-lake/latest/APIReference/API_LakeConfigurationResponse.html